src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -217,6 +219,16 @@ def _build_basic_constraints(self, ext):
 
         return x509.BasicConstraints(ca, path_length)
 
+    def _build_subject_key_identifier(self, ext):
+        asn1_string = self._backend._lib.X509V3_EXT_d2i(ext)
+        assert asn1_string != self._backend._ffi.NULL
+        asn1_string = self._backend._ffi.cast(
+            "ASN1_OCTET_STRING *", asn1_string
+        )
+        return x509.SubjectKeyIdentifier(
+            self._backend._ffi.buffer(asn1_string.data, asn1_string.length)[:]
```

Comment:
`[:]` is going to copy the buffer. Do you not need to free the original data or is that handled by the extension instance?